templates/common/render/render_test.go

```diff
@@ -1287,11 +1347,13 @@ steps:
 				t.Errorf("scratch directory contents were not as expected (-got,+want): %s", diff)
 			}
 
-			gotDestContents := abctestutil.LoadDir(t, outDir)
+			gotDestContents := abctestutil.LoadDir(t, outDir, abctestutil.SkipGlob(".abc/manifest*"))
```

Comment:
nits: Add another comment to explain that manifest will be evaluated separately for the reason XXXX?

Reply:
Done.